### PR TITLE
Adding contributor story for Peeyush

### PR DESCRIPTION
**Pull request by Pensu**
This is an entry for the contributor story series. 

CC @mbbroberg

### CODE REVIEW

File: content/en/blog/2021/peeyush-contributor-story.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+I loved the experience of working with the community and it made me want to contribute more. I also wanted to help other contributors who are looking to get started with Kubernetes. I came across SIG ContribEx (Contributor Experience), whose purpose was to help contributors make the most of the Kubernetes community. It felt like the right place to be. But since I was in the IST time zone and most of the meetings were in the US time zone, it was a little difficult for me to attend those meetings. Luckily, at the same time, ContribEx was planning on an APAC-specific time meeting and I started attending those meetings. Today, I actively participate in two sub-projects; APAC coordinators, to help contributors in the APAC region; and contributor-comms, to improve communication for contributors.
+
+## Finding my place to contribute
+
+CNCF is home for Kubernetes and a lot of other cloud native projects. Once I got involved with Kubernetes and attended a couple of KubeCons, it made me aware of those projects as well. One of those projects is OpenEBS, a leading cloud native storage solution. I liked the project and started contributing and became a maintainer.
```
Comment (mbbroberg) [inline content/en/blog/2021/peeyush-contributor-story.md:22]:
```suggestion
The CNCF is home for Kubernetes and a lot of other cloud native projects. Once I got involved with Kubernetes and attended a couple of KubeCons, it made me aware of those projects as well. One of those projects is OpenEBS, a leading cloud native storage solution. I liked the project and started contributing and became a maintainer.
```

File: content/en/blog/2021/peeyush-contributor-story.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+layout: blog
+title: "From Kubernetes for work to Kubernetes for play" 
+date: 2021-05-07
+Slug: contributor-stories-series
```
Comment (mbbroberg) [inline content/en/blog/2021/peeyush-contributor-story.md:5]:
```suggestion
date: 2021-05-14
Slug: kubernetes-work-and-play
```

**Review by mbbroberg — COMMENTED** [review]
Looking good @Pensu! Two minor edits that I hope we can work through in today's meeting and get this whole series published.

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

/hold
please cancel when ready

**Pull request merged**